src/main/java/com/innoq/spring/boot/actuate/health/HealthDetailEnhancer.java

```diff
@@ -33,7 +33,7 @@
  */
 public final class HealthDetailEnhancer implements HealthIndicator {
 
-    private final HealthIndicator indicator;
+    final private HealthIndicator indicator;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)